### PR TITLE
Ubuntu 20.04 fixes for CMake discovery plugin

### PR DESCRIPTION
**Pull request by tdenewiler**
Fixes to CMake flags and template to get CMake discovery to work on Ubuntu 20.04. This was blocking any tools that depend on CMake or Make running.

### CODE REVIEW

File: statick_tool/plugins/discovery/cmake_discovery_plugin.py [hunk 1]
```diff
@@ -52,6 +52,7 @@ def scan(
             "cmake",
             ".",
             "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+            "-DBUILD_GMOCK=ON",
```
Comment (xydesa) [inline statick_tool/plugins/discovery/cmake_discovery_plugin.py:55]: not as part of this fix, but in the future it'd be good to extract these options into the statick config file.

**Comment by xydesa** [comment]
It'd be nice if 20.04 were tested via github actions just like 16.04 and 18.04

File: statick_tool/plugins/discovery/cmake_discovery_plugin.py (hunk 1 above)
Comment (tdenewiler) [reply statick_tool/plugins/discovery/cmake_discovery_plugin.py:55]: I agree. I created #250 to remind me.

**Comment by tdenewiler** [comment]
Reply to https://github.com/sscpac/statick/pull/249#issuecomment-672267516:

I made #251 to track adding the `ubuntu-20.04` tag. It should be pretty easy, unless tests fail...

**Pull request merged**